packages/bpk-component-slider/src/bpk-slider.scss

```diff
@@ -0,0 +1,89 @@
+@import '~bpk-mixins';
+
+$slider-track-size: 6 * $bpk-one-pixel-rem;
+$slider-control-size: 30 * $bpk-one-pixel-rem;
+$slider-control-inset-size: 20 * $bpk-one-pixel-rem;
+$slider-control-size-large: 42 * $bpk-one-pixel-rem;
+$slider-control-inset-size-large: 30 * $bpk-one-pixel-rem;
+
+.bpk-slider {
+  margin-bottom: $bpk-spacing-xxl;
+
+  &__range {
+    // eslint-disable-next-line selector-attribute-quotes
+    div[class$='1'] {
+      background: $bpk-color-blue-500;
+    }
+  }
+
+  &__simple {
+    // eslint-disable-next-line selector-attribute-quotes
+    div[class$='0'] {
+      background: $bpk-color-blue-500;
+    }
+  }
+
+  &__value {
+    margin-bottom: $bpk-spacing-md;
+
+    &--min {
+      display: inline-block;
+    }
+
+    &--max {
+      display: inline-block;
+    }
+  }
+}
+
+.bpk-slider-handle {
+  width: $slider-control-size;
+  height: $slider-control-size;
+  border: $bpk-one-pixel-rem solid $bpk-color-gray-100;
+  border-radius: $slider-control-size;
+  background: $bpk-color-white;
+  box-shadow: $bpk-box-shadow-sm;
+  cursor: pointer;
+  vertical-align: top;
+
+  &::after {
+    content: '';
+    display: block;
+    width: $slider-control-inset-size;
+    height: $slider-control-inset-size;
+    margin-top: 4 * $bpk-one-pixel-rem;
+    margin-left: 4 * $bpk-one-pixel-rem;
+    border-radius: 38 * $bpk-one-pixel-rem;
+    box-shadow: inset 0 0 0 $bpk-one-pixel-rem $bpk-color-gray-200;
+  }
+
+  &--active {
+    transform: scale(1.2) translate3d(0, 0, 0);
+  }
+
+  &--large {
+    top: -5 * $bpk-one-pixel-rem;
+    width: $slider-control-size-large;
+    height: $slider-control-size-large;
+    border-radius: $slider-control-size-large / 2;
+
+    &::after {
+      width: $slider-control-inset-size-large;
+      height: $slider-control-inset-size-large;
+      margin-top: 5 * $bpk-one-pixel-rem;
+      margin-left: 5 * $bpk-one-pixel-rem;
+    }
+  }
+}
+
+.bpk-slider-bar {
+  position: relative;
+  top: $slider-control-size / 2.5;
+  height: $slider-track-size;
+  border-radius: $slider-track-size / 2;
+  background-color: $bpk-color-gray-100;
+}
+
+.bpk-slider-wrapper {
+  margin-bottom: $bpk-spacing-lg;
```

Comment:
I would avoid imposing margin bottom to all instances of a slider - let userland control this behaviour.

Reply:
Cool, I have removed it